src/main/java/com/dataguise/saas/DataOrientedSaasUiApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.dataguise.saas;
+
+import org.apache.log4j.Logger;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class DataOrientedSaasUiApplication {
+    Logger logger = Logger.getLogger(DataOrientedSaasUiApplication.class);
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.design.VisibilityModifierCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/DataOrientedSaasUiApplication.java L9
    
    Variable &#39;logger&#39; must be private and have accessor methods.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a987283837>*